src/api/java/com/minecolonies/coremod/colony/permissions/Action.java

```diff
@@ -0,0 +1,36 @@
+package com.minecolonies.coremod.colony.permissions;
```

Comment:
I think this belong to a different pr, and could be merge well before dman 2.0 is ready